homeassistant/components/homekit/type_covers.py

```diff
@@ -20,14 +20,11 @@ class WindowCovering(HomeAccessory):
     The cover entity must support: set_cover_position.
     """
 
-    def __init__(self, hass, entity_id, display_name, **kwargs):
+    def __init__(self, entity_id, display_name, **kwargs):
         """Initialize a WindowCovering accessory object."""
         super().__init__(display_name, entity_id,
                          CATEGORY_WINDOW_COVERING, **kwargs)
```

Comment:
If you call category with a keyword it also protects against future changes

 ```super().__init__(name, entity_id, category=CATEGORY_*, **kwargs)```

*same for all HomeAccessories